### ISSUE TITLE
ol.interaction.Interaction API docs typo

### ISSUE THREAD

**Issue by drnextgis** [openlayers/ol3#5059]
`Required` is doubled [here](http://openlayers.org/en/v3.14.2/apidoc/ol.interaction.Interaction.html):

```
Method called by the map to notify the interaction that a browser event was dispatched to the map.
The function may return false to prevent the propagation of the event to other interactions in the map's interactions chain.
Required. Required.
```

**Comment by drnextgis** [comment]
PR https://github.com/openlayers/ol3/pull/5061

**Comment by marcjansen** [comment]
Closed via #5061
